src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -911,6 +921,9 @@ class HookUtils(object):
         Handler for execjob_attach events.
         """
         pbs.logmsg(pbs.EVENT_DEBUG4, "%s: Method called" % caller_name())
+        # delete new job marker files here in case hook events before me
```

Comment:
You are no longer looking for "*.new" files. Please update the comment.

Reply:
Fixed.